include/roboptim/core/portability.hh

```diff
@@ -83,8 +83,8 @@
 
 // Required to avoid size_t resolution error with MSVC. Triggered by
 // the boost/tuple/tuple_io.hpp inclusion in roboptim/core/io.hh.
-#ifdef _WIN32
-# define BOOST_NO_STD_LOCALE
-#endif
+//#ifdef _WIN32
+//# define BOOST_NO_STD_LOCALE
+//#endif
```

Comment:
Sorry, I was sure I removed it.
This was added by Thomas in 2010. I could only test with Visual 2013 and 2015. Maybe @francois-keith can tell us if the current version works with Visual 2012, to decide if we remove it completely or if we need some ifdef